product/toolbars/containers_center_tb.yaml

```diff
@@ -44,4 +44,17 @@
       :title: 'Edit Tags for this #{ui_lookup(:table=>"container")}'
       :url_parms: 'main_div'
       :enabled: 'false'
-      :onwhen: '1+'
\ No newline at end of file
+      :onwhen: '1+'
+- :name: container_monitoring
+  :items:
+  - :buttonSelect: container_monitoring_choice
+    :image: monitoring
+    :title: Monitoring
+    :text: Monitoring
+    :items:
+    - :button: container_perf
+      :image: capacity
+      :text: "Utilization"
+      :title: "Show Capacity & Utilization data for this Container"
+#     :url: '/show'

```

Comment:
needs removal? (commented code)
does adding this option to this yaml work ok in the containers explorer? (there are some issues with the tagging button, for example)